question/templates/setting_qbanksetting.mustache

```diff
@@ -0,0 +1,42 @@
+{{!
+    This file is part of Moodle - http://moodle.org/
+
+    Moodle is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Moodle is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+}}
+{{!
+    @template core_admin/setting_qbanksetting.mustache
+
+    Admin question bank setting template.
+
+    Context variables required for this template:
+    * name - plugin name
+
+    Example context (json):
+    {
+        "name": "createcourse",
+    }
+}}
+
+<ul class="list list-group" id="sortable">
+    {{#name}}
+        <li class="item list-group-item list-group-item-action" draggable="true">
+            <div class="fa fa-arrows">
```

Comment:
Probably the best way to call the arrows will be like this {{> core/drag_handle}}

Reply:
Fixed, working on sortable ist now